src/ResourceManager/Compute.ManagedService/Commands.Compute.ManagedService/ConvertTo-AzureRmVhd.psm1

```diff
@@ -0,0 +1,35 @@
+# Root module manifest for module 'AzureRM.ManagedComputeServices'
+
+$PSDefaultParameterValues.Clear()
+Set-StrictMode -Version Latest
+
+$module = Get-Module AzureRM.Profile 
+if ($module -ne $null -and $module.Version.ToString().CompareTo("4.0.0") -lt 0) 
+{ 
+    Write-Error "This module requires AzureRM.Profile version 4.0.0. An earlier version of AzureRM.Profile is imported in the current PowerShell session. Please open a new session before importing this module. This error could indicate that multiple incompatible versions of the Azure PowerShell cmdlets are installed on your system. Please see https://aka.ms/azps-version-error for troubleshooting information." -ErrorAction Stop 
+} 
+elseif ($module -eq $null) 
+{ 
+    Import-Module AzureRM.Profile -MinimumVersion 4.0.0 -Scope Global 
+}
+Import-Module (Join-Path -Path $PSScriptRoot -ChildPath Microsoft.Azure.Commands.ManagedComputeServices.dll)
+
+
+
+$module = Get-Module Hyper-V;
+if ($module -ne $null -and $module.Version.ToString().CompareTo("1.1") -lt 0)
```

Comment:
You should add a couple of lines like this

```powershell
$installedModule = Get-Module -ListAvailable Hyper-V | Select -First 1
```

and then add a test in this first if to see if this module is on the machine at all. i.e.

```powershell
if ($installedModule -eq $null -or ($module -ne $null -and $module.Version.ToString().CompareTo("1.1") -lt 0))
```